blackbox/docs/sql/reference/alter_table.txt

```diff
@@ -112,3 +88,93 @@ columns with a value each to identify the partition to alter.
 :value: The columns value.
 
 .. SEEALSO:: :ref:`Alter Partitioned Tables <partitioned_tables_alter>`
+
+
+Arguments
+=========
```

Comment:
imho these are not arguments but sub comands

Reply:
@seut sql server list does also as *Arguments* (see https://docs.microsoft.com/en-us/sql/t-sql/statements/alter-table-transact-sql#arguments)